js/languagePickerView.js

```diff
@@ -22,7 +22,8 @@ define([
             var data = this.model.toJSON();
             var template = Handlebars.templates[this.constructor.template];
             this.$el.html(template(data));
```

Comment:
i would add the class in the template ... since ``this.$el.add(data._classes)` is a function call and an extra update for the Dom.